frontend/src/pages/org/crawls-list.ts

```diff
@@ -209,13 +212,19 @@ export class CrawlsList extends LiteElement {
             ${when(
               this.isCrawler,
               () => html`
-                <sl-button
-                  size="small"
-                  @click=${() => (this.isUploadingArchive = true)}
+                <sl-tooltip
+                  content=${msg("Org Storage Full")}
+                  ?disabled=${this.orgStorageQuotaReached === false}
```

Comment:
It seems like this has the opposite effect of disabling the button when the storage quota is not reached.
```suggestion
                  ?disabled=${this.orgStorageQuotaReached}
```

Reply:
The tooltip and the button have reverse disabled logic. We want the button to be disabled if the quota is reached, so that the user can't upload a WACZ. But we want the tooltip disabled if the quota *isn't* yet reached, so that it only displays when the quota is reached. Took me a second to get my head around.

Reply:
Gotcha!